client/my-sites/patterns/components/category-not-found/index.tsx

```diff
@@ -8,16 +8,27 @@ import './style.scss';
 
 export const PatternsCategoryNotFound = () => {
 	const isLoggedIn = useSelector( isUserLoggedIn );
-	const translate_not_yet = useTranslate();
-
-	const comment =
-		'Message displayed when an invalid category was specified while searching block patterns';
+	const translate = useTranslate();
 
+	const emptyContentStrings = {
+		title: translate( "Oops! We can't find this category!", {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+		line: translate( "The category you are looking for doesn't exist.", {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+		action: translate( 'Browse all patterns', {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+	};
```

Comment:
Since we must add those comments anyway, maybe we should make them a bit more specific?

```suggestion
	const emptyContentStrings = {
		title: translate( "Oops! We can't find this category!", {
			comment:
				'Heading displayed when an invalid category was specified while searching block patterns',
		} ),
		line: translate( "The category you are looking for doesn't exist.", {
			comment:
				'Message displayed when an invalid category was specified while searching block patterns',
		} ),
		action: translate( 'Browse all patterns', {
			comment:
				'Label of a button displayed when an invalid category was specified while searching block patterns',
		} ),
	};
```

Reply:
By the way, here is another approach we could consider that gets rid of one level of indentation:

```javascript
const title = translate( "Oops! We can't find this category!", {
  comment:
    'Message displayed when an invalid category was specified while searching block patterns',
} );

const line = translate( "The category you are looking for doesn't exist.", {
  comment:
    'Message displayed when an invalid category was specified while searching block patterns',
} );

const action = translate( 'Browse all patterns', {
  comment:
    'Message displayed when an invalid category was specified while searching block patterns',
} );

return (
  <EmptyContent
    title={ title }
    line={ line }
    action={ action }
```

Reply:
I like both of these suggestions... the more specific comments but also the individual variables in favor of the single object. Thanks!